fhir-server/src/test/java/au/csiro/pathling/fhirpath/parser/ParserTest.java

```diff
@@ -518,12 +515,22 @@ void testTranslateWithWhereAndTranslate() {
         .hasRows(spark, "responses/ParserTest/testTranslateWithWhereAndTranslate.csv");
   }
 
+  @Test
+  public void testCombineOperator() {
+    assertThatResultOf("name.family combine name.given")
+        .isElementPath(StringPath.class)
+        .selectResult()
+        .hasRows(spark, "responses/ParserTest/testCombineOperator.csv");
+  }
+
   @Test
```

Comment:
Add a test for <element> combine <literal> and maybe for combining resources.

Reply:
Fixed in eb0a0c5428d450d818940d453a4274e1c2358b1b and 5610e894a23b60c0b2e3143109c7006d9a5022d8.